reference/service/saml-aad.md

```diff
@@ -0,0 +1,111 @@
+  select **SAML**.
+
+    ![Single sign-on settings](../../images/reference/service/saml-aad/single-sign-on.png)
+
+1. Click the **Edit** icon on the **Basic SAML Configuration** panel.
+
+    ![SAML configuration](../../images/reference/service/saml-aad/saml-configuration.png)
+
+The next step is to enter Pulumi-specific configuration data into your Azure AD application.
+The values you will want to use are dependent upon your organization's name on Pulumi. Be sure
+to replace `acmecorp` with whatever your specific Pulumi organization's name is.
+
+<style>
+td, th {
+    padding: 8px 8px;
+    border: 1px solid rgba(0,0,0,0.13);
+}
+
+thead tr th {
+    color: #00acf2;  /* $primary2, blue */
+    font-weight: 800;
+}
+
+tbody tr td {
+    padding-left: 16px;
+    padding-right: 16px;
+}
+</style>
+
+| SAML Setting | Value |
+| --------------- | ----- |
+| Identifier (Entity ID) | https://api.pulumi.com/login/**acmecorp**/sso/saml/metadata |
+| Replay URL | https://api.pulumi.com/login/**acmecorp**/sso/saml/acs |
+
+
+<p><!-- space between table and image --></p>
+
+![Edited SAML configuration](../../images/reference/service/saml-aad/edited-saml-configuration.png)
+
+Now the Azure AD-side of the SAML SSO configuration is complete. The next and final
+step is to configure the Pulumi Cloud Console to receive SAML SSO requests from your
+Azure AD.
+
+> **NOTE:** Be sure to assign users and groups to use your new _Pulumi Cloud Console_ SAML application.
+> That is how you can control access to who can join your Pulumi organization. See the
+> [Azure AD documentation](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/configure-single-sign-on-non-gallery-applications#assign-users-and-groups-to-your-saml-application)
+> for more information.
+
+## Configuring Your Pulumi Organization
+
+To configure your Pulumi organization to accept SAML SSO requests from Azure AD, you will need to
+download the SAML application's configuration data and then pass that to Pulumi.
+
+1. Back on the Azure AD's application settings page, click the **SAML Signing Certificate** panel.
+  Then click **Download** next to **Federated Metadata XML** and save the resulting file.
+
+    ![Download XML](../../images/reference/service/saml-aad/download-xml.png)
+
+1. Sign into the Pulumi Cloud Console and navigate to your SAML organization. Navigate to the
+  **Settings** tab and then select **SAML SSO**.
+
+1. Open up the XML document you downloaded from the Azure AD portal, and paste its full contents
+  into the **Identity Provider Metadata** field.
+
+    ![Provide the XML IdP descriptor](../../images/reference/service/saml-aad/pulumi-saml-settings-page.png)
+
+1. Click **Save**.
+
+## Signing into Pulumi using Azure AD
+
+Once your Azure AD application is created, and its configuration data passed to Pulumi, you can now
+log into the Pulumi Cloud Console using your SAML SSO credentials.
```

Comment:
Bit of a nit, but we should probably choose either "sign in" or "log in" for this sort of thing. Personally I'm partial to "sign in" (and use `login` more as a noun -- e.g., your login === your username), as it's a bit friendlier/more colloquial, but could go either way -- I just think we should be consistent. E.g.:

&times; Please log in
&times; Please login
&times; Please signin
&check; Please sign in

Reply:
Sounds good to me. I never know which to use, so I'll take a quick pass and try to standardize on "sign in".